ext/hash/hash.c

```diff
@@ -1468,6 +1485,14 @@ PHP_METHOD(HashContext, __serialize)
 	Z_TRY_ADDREF(tmp);
 	zend_hash_next_index_insert(Z_ARRVAL_P(return_value), &tmp);
 
+	if (hash->args) {
+		ZVAL_ARR(&tmp, hash->args);
+		Z_ADDREF(tmp);
```

Comment:
The array might be in SHM, so this should use Z_TRY_ADDREF.

Reply:
Yeah, that's where I was scratching my head as well whether a try would be useful. Thanks, fixed.